agreementbot/secure_api.go

```diff
@@ -1053,14 +1053,15 @@ type SecretRequestInfo struct {
 //     description: "The secret key (name)."
 //
 // responses:
-//   '400':
```

Comment:
Is there a way to exclude the change swagger format? I found that after the formatting, it will not generate the correct swagger page